src/CommandOptions.mjs

```diff
@@ -25,6 +25,7 @@ export const makeConfig = (argv) => {
     .option('--depth <depth specifier>', 'See the description above.', '1')
     .option('--css <paths>', 'The comma seprated list of css file paths to include.  The paths must be accesible from the current directory.', commaSeparatedList)
     .option('--no-strictMode', 'Turns off the strict mode.')
+    .option('--titlePage <title string>', 'Sets title page string.', 'Titlepage')
```

Comment:
Maybe 'Sets title page **_toc_** string' is clearer?  English is not my first language so I am not sure if it is correct though.

Reply:
The toc title is set by `:toc-title:` attribute in the `*.doc`. I was specifically getting at the word `Titlepage` appearing just underneath the toc title. See attached screenshot.
![Screen Shot 2021-04-27 at 11 01 15 ](https://user-images.githubusercontent.com/5176962/116264690-81acc700-a748-11eb-8f1b-bbf781169f5e.png)

Reply:
Yes, I know what you mean as I read the your patch.  But when I first read the descriptions in `CommandOptions.mjs`, I thought you implemented the way to customize the main title or some kind of title page contents from the command line.  So I checked the patch and found out that you wanted to customize the toc item for the  top page link.   I don't know any better ways to describe in English than as I suggested in the previous comment.

Reply:
Okay. Yes, your wording could help clarify its use. I can't think of an alternative.

Reply:
Done.